modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/spi/searcher/SXPBlueprintSearchRequestContributor.java

```diff
@@ -69,16 +69,16 @@ private void _contributeSXPBlueprintExternalReferenceCode(
 					object);
 		}
 
-		if (object == null) {
-		}
-		else if (object instanceof String) {
+		if (object instanceof String) {
 			String string = (String)object;
 
 			if (Validator.isNotNull(string)) {
```

Comment:
Generally, in these kind of places, I'd prefer Validator.isBlank() instead of Validator.isNotNull(). Theoretically, if the string/ERC is 'NULL', which is a valid ERC, this will fail.